sdk/keyvault/keyvault-admin/package.json

```diff
@@ -110,19 +110,19 @@
     "@azure/core-lro": "^2.2.0",
     "@azure/core-paging": "^1.1.1",
     "@azure/core-rest-pipeline": "^1.1.0",
-    "@azure/core-tracing": "1.0.0-preview.13",
```

Comment:
don't we need to declare a new min version of `core-client` or `core-http` to make sure we can get the new stuff?

Reply:
Yes for sure. But I am kinda over having packages depend on unreleased core packages 😄 so my plan is to use keyvault for testing this, but revert those changes before taking this PR out of draft. Once these changes are released I'll bump the minimum version and remove the dependency on core-tracing in keyvault in a separate PR (next month)